duckduckgo/iOS
# Remove duplicate NetP notifications

Task/Issue URL: https://app.asana.com/0/0/1205617766971743/f

⚠️ This PR is stacked on #2053 until it’s merged. ⚠️

**Description**:

On introducing notifications to NetP, I noticed that they were a big noisy. This PR can be merged to deal with that if we agree.

**Steps to test this PR**:
1. Launch the app, go to Settings -> Debug and ensure you are set as an Internal User
2. Go to Settings -> Network Protection and make sure the VPN is running
3. Go to Settings -> Debug -> Network Protection and select Test Notification. Spam it a few times
4. **Observe there is only one notification at the top of the screen**
5. Also select Connection Interruption and wait for it and then the reconnection notification to show. Try this a few times.
6. **Observer they don’t build up either**

N.B: I noticed that using the interruption simulation / test notification ends up with the notifications not actually appearing in the app notifications. I’m assuming this is because the app is open. However, on testing this and receiving interruption / reconnection notifications for real that they successfully appeared on the notification menu and were never duplicated. This is possibly my Friday brain forgetting the default notification behaviour, but I can assure you this works as expected.

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


—
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
